code/modules/reagents/chemistry/reagents/alcohol.dm

```diff
@@ -1779,3 +1779,25 @@
 	mutated = TRUE
 
 	return ..()
+
+/datum/reagent/consumable/ethanol/voxwine
+	name = "Vox Wine"
+	id = "voxwine"
+	description = "An interesting drink some may find rather offensive."
+	color = "#cdd37b" // rgb: 205, 211, 123
+	alcohol_perc = 0.2
```

Comment:
It's made from straight ethanol, this would make more sense if was higher to me. Up to you though